docs/design/rule-manager/sequence-diagram.puml

```diff
@@ -0,0 +1,25 @@
+@startuml
+title Rule manager sequence diagram
```

Comment:
This one looks weird as for me, you are mixing together 2 workflows here, one for flow hs ops and one for switch manager. I'd split it

Reply:
Added separators with operation descriptions.